playbooks/roles/bats/tasks/main.yml

```diff
@@ -1,4 +1,9 @@
 ---
+- name: "Install git"
+  yum:
+    name: "git"
+    state: "installed"
```

Comment:
:+1:

We need to update this role to allow testing updates to bats from a PR or local environment. I'll file an issue.